pkg/cover/report.go

```diff
@@ -180,15 +199,9 @@ func (rg *ReportGenerator) lazySymbolize(progs []Prog) error {
 		return err
 	}
 	rg.Frames = append(rg.Frames, frames...)
-	uniqueFrames := make(map[uint64]bool)
-	var finalFrames []backend.Frame
-	for _, frame := range rg.Frames {
-		if !uniqueFrames[frame.PC] {
-			uniqueFrames[frame.PC] = true
-			finalFrames = append(finalFrames, frame)
-		}
-	}
-	rg.Frames = finalFrames
```

Comment:
What was the motivation to remove this piece of code?

Reply:
1 pc can be decoded into multiple frames if the pc points to some inline function.
So previous code will wrong filter out such kind of frame.